crypto/asn1/a_time.c

```diff
@@ -147,10 +147,9 @@ ASN1_GENERALIZEDTIME *ASN1_TIME_to_generalizedtime(ASN1_TIME *t,
     if (!out || !*out) {
         if (!(ret = ASN1_GENERALIZEDTIME_new()))
             goto err;
-        if (out)
-            *out = ret;
-    } else
+    } else {
         ret = *out;
+    }
```

Comment:
Yup that was my point, but having the braces there is not harmful, so I really don't care. The other style issues were NULL checks that were being treated as boolean, but that's outside the scope of this.